mmcv/engine/test.py

```diff
@@ -12,7 +12,7 @@
 from mmcv.runner import get_dist_info
 
 
-def single_gpu_test(model, data_loader):
+def single_gpu_test(model, data_loader) -> list:
```

Comment:
Please add a type hint for `data_loader`.

Reply:
> Please add a type hint for `data_loader`.

@zhouzaida Hello, l have add a type hint for all arguments